storybook/preview-head-vrt.html

```diff
@@ -1,3 +1,17 @@
+<script>
+	// Flag the document as in VRT mode.
+	document.documentElement.dataset.vrt = true;
+
+	window.addEventListener( 'load', async () => {
+		// Wait for all web fonts to be loaded.
+		const readyPromises = [ ...document.fonts.values() ].map( ( font ) => font.load() );
+
+		await Promise.allSettled( readyPromises );
+
+		// Signal Backstop that the environment is ready to go.
+		document.body.classList.add( 'backstopjs-ready' );
+	} );
```

Comment:
This has been moved from the general `preview-head.html` to this VRT-specific file that we added later and is obviously more appropriate.